schema.graphql

```diff
@@ -67,3 +48,35 @@ type UniquesTransfer @entity {
   timestamp: Float @index
   success: Boolean
 }
+
+type Pot @entity {
+  id: ID!
+  potId: Float! @index 
+  owner: String! @index
+  feeQuota: BigInt!
+  reserveQuota: BigInt!
+  sponsorshipType: String!
+  sponsoredAccounts: [PotBalance] @derivedFrom(field: "pot")
+  createdAt: Float @index
+  updatedAt: Float @index
```

Comment:
Are we using these fields in any of our queries?

Reply:
Yes. We use it for sorting, also in the uniques creation for example the items is minted but metadata could be set up later or change it ownership